packages/compass-import-export/src/modules/import.ts

```diff
@@ -197,6 +199,18 @@ async function getErrorLogPath(fileName: string) {
   return path.join(importErrorLogsPath, errorLogFileName);
 }
 
+async function signalBloatedDocuments(appRegistry: AppRegistry) {
+  const instanceStore: any = appRegistry.getStore('App.InstanceStore');
+  if (instanceStore && instanceStore.fetchCollectionMetadata) {
+    const collMetadata = await instanceStore.fetchCollectionMetadata(ns);
+    showBloatedDocumentSignalToast({
+      onReviewDocumentsClick: () => {
+        appRegistry.emit('open-namespace-in-new-tab', collMetadata);
+      },
+    });
+  }
```

Comment:
This should be done in the instance store where all the other metadata resolution is currently handled